frontend/test/pytest/test_template.py

```diff
@@ -17,12 +17,35 @@
 import numpy as np
 import pennylane as qml
 import pytest
-from pennylane import numpy as pnp
 from scipy.stats import norm
 
 from catalyst import for_loop, qjit
 
 
+def test_adder(backend):
```

Comment:
You can ask pylint to disable too-many-lines, see for example https://github.com/PennyLaneAI/catalyst/blob/3fac9c2a117fa9a48df5694419d4883f96dd71b6/frontend/catalyst/jax_tracer.py#L102